guide/onboarding/creating-a-new-wallet.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: guide
+title: Creating a new a wallet
+description: Steps to take when creating a new wallet
+parent: Onboarding
+nav_order: 2
+permalink: /guide/onboarding/creating-a-new-wallet/
+main_classes: -no-top-padding
+---
+
+
+# What is a bitcoin wallet?
+
+First, bitcoin [wallets](https://bitcoin.design/guide/glossary/#wallet) are not analogous to their namesakes. Physical wallets are places to store physical currency, credit cards, IDs, and so on. Bitcoin wallets don't actually store bitcoin.
+
+Instead, a bitcoin wallet contains the "private keys" to your bitcoin, the bitcoin is located on a network that contains a communally sustained public ledger with a record of everyone's transactions and balances (so think of bitcoin wallets more like keychains). Like regular keys on a keychain, these keys can be copied. For example, if you gave someone a copy of your house keys, they could get into your home. Likewise, if someone has a copy of your private key, they can access your funds. This makes private key management one of the most important considerations of any bitcoin owner.
+
+There are a few different types of wallets that let you send, receive, and store bitcoin.
+
+{% include /tip/open.html label="Tip: Break onboarding up" icon="info" color="blue" %}
+
+The wallet creation process is an excellent place to break things up into different activities, events, and tasks. For example, you might want to dedicate multiple screens to generate a recovery phrase and explain why it is essential.
+
+{% include /tip/close.html %}
+
+# Creating a new wallet
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/onboarding/intro-screen.jpg"
+   retina = "/assets/images/guide/onboarding/intro-screen@2x.jpg"
+   alt-text = "Onboarding intro screen"
+   layout = "float-left-desktop -background -shadow"
+   caption = "From the [Onboarding Prototype](https://www.figma.com/proto/oLJlzjXqgoU7efgXJM8FlK/Onboarding-Prototypes?node-id=1%3A4&viewport=332%2C345%2C0.5653080344200134&scaling=min-zoom&page-id=0%3A1)"
```

Comment:
The prototype doesn't work for me. Looks like this on desktop.

<img width="648" alt="image" src="https://user-images.githubusercontent.com/695901/114571873-4786f400-9c77-11eb-9369-7fc4d7ed7dc0.png">

Reply:
Same here thought it was my adblock or something.

Reply:
🚧